### ISSUE TITLE
SDL Performance Issue

### ISSUE THREAD

**Issue by jvernet** [86Box/86Box#964]
**Describe the bug**
Since some weeks, I'm experiencing perfomances issue, running 86Box with SDL Hardware renderer: My GPU (GTX 960OC) is 100% high in 3D, fan running loud. Even emulating a 8086/10 I have only 64% percent displayed in the status bar, even only on a DOS screen with only a c:>prompt

Using second screen on the integrated Intel GPU is worse (less than 54% percent).

i thought that my own build have a problème, but using Regular Build form github does the same.

**To Reproduce**
Start any machine,select SDL Hardware Renderer, running any version of DOS

**Desktop (please complete the following information):**
 - OS: Windows 10 64
 - 86Box version: latest build from Github, I thinj that it was working somes monts/Week ago?
 - Build type: regular or my own build.
cPU: i4790 Core i5, NVIDIA GTX960 OC -didn't try to play with NVIDIA parameters

**Comment by OBattler** [comment]
Did you try the SDL Software renderer? Though I'm not quite sure why it's doing this, it works fine on my end. Could you bisecet and see what build introduced the problem?

**Comment by PentiumMMX166** [comment]
It does have worse performance than PCem, PCem runs smooth with a Pentium 75, 16 MB RAM, S3 Trio 64, Windows 95, when i try it in 86box with the same settings, i get tons of lag and stutter and it hangs around 60% on the window's title bar. The PC that i'm using is a AMD Athlon II X4 640 OC 3.2 GHz, 8 GB (2x 4gb DDR2-800) RAM, GTX 660, Windows 7 Ultimate.

**Comment by PentiumMMX166** [comment]
yes, same results.

**Comment by jvernet** [comment]
SDL Software Renderer is better.
What is curious is that on a Latptop with only the Intel GPU, it run better...

**Comment by OBattler** [comment]
I heard reports that 86Box performance has now gone up since August, both of you please see if that's the case.

**Comment by jvernet** [comment]
Hello,
Nothing new on performance side. Using SDL Hardware renderer, I only got 50/60% perf in the tittle bar, big lag, audio corrupted. Heavy GPU usage (100% !!!).
It's working with Software renderer, lacking the antialiasing effect, so it's weird in 2x display.
My configuration: W10 2004, GTX 960. I tried to change mysylf parameters in win_sdl.c without success.
PCEM if fine, even wen activating shaders.

How do I enable SDL logging, to see wich renderer is used ?
